src/consensus/consensus.h

```diff
@@ -17,8 +17,12 @@ static const unsigned int MAX_BLOCK_BASE_SIZE = 1000000;
 /** The maximum allowed number of signature check operations in a block (network rule) */
 static const int64_t MAX_BLOCK_SIGOPS_COST = 80000;
 
+/** The minimum allowed size for a transaction */
+static const unsigned int MIN_TRANSACTION_BASE_SIZE = 10;
```

Comment:
I don't think there's such thing as a 10 byte transaction.   Probably, after segwit, this should be 30?